src/observeProps.js

```diff
@@ -1,12 +1,22 @@
 import React, { PureComponent } from 'react';
-import { Observable, Observer, Subject } from 'rx-lite';
+
+import { Subject } from 'rxjs/internal/Subject';
+import { from } from 'rxjs/internal/observable/from';
+
+import { map } from 'rxjs/internal/operators/map';
+import { filter } from 'rxjs/internal/operators/filter';
+import { distinctUntilChanged } from 'rxjs/internal/operators/distinctUntilChanged';
+import { endWith } from 'rxjs/internal/operators/endWith';
 
 import { createSelectorForFunction, throwError } from './utils';
 import { IS_PROP_LISTENER, PROP_KEY } from './constants';
 
 
 export default function watchProps(TargetComponent) {
   return class WatchPropsWrapper extends PureComponent {
+    /**
```

Comment:
Invalid comment. Don't think it's needed at all. What do you think?

Reply:
@McReader it is invalid. Agree that it's redundant. Removed.